tests/misc/test_basic_without_ssh.py

```diff
@@ -25,11 +25,11 @@ def existing_shared_sr(host):
     assert sr is not None, "A shared SR on the pool is required"
     return sr
 
-@pytest.mark.incremental # tests depend on each other. If one test fails, don't execute the others
 @pytest.mark.multi_vms # run them on a variety of VMs
 @pytest.mark.big_vm # and also on a really big VM ideally
 class TestBasicNoSSH:
-    def test_start(self, imported_vm):
+    @pytest.fixture(scope="class", autouse=True)
+    def vm_start(self, imported_vm):
```

Comment:
Ok, here too `test_start` WAS a real test, but it's a good thing to get rid of `@pytest.mark.incremental`, so ok for removing both the start and stop tests.

Reply:
restored as described above